cfgov/tccp/jinja2/tccp/includes/card_list.html

```diff
@@ -41,57 +41,73 @@
 {{ data_published(stats_all.first_report_date) }}
 {%- endif %}
 
-{%- macro card_name_cell(card) -%}
-    <a href="{{ card.url }}">
-        <div class="a-card-institution-name">{{ card.institution_name }}</div>
-        <div>{{ card.product_name }}</div>
-    </a>
-{%- endmacro -%}
-
-{%- macro purchase_apr_cell(card) -%}
+{%- macro card_rating(card) -%}
     {%- set adjective = purchase_apr_adjectives[
         card.purchase_apr_for_tier_rating
     ] -%}
 
-    {{- apr(card.purchase_apr_for_tier) -}}
-
-    <span class="a-card-purchase-apr-rating a-card-purchase-apr-rating__{{ adjective }}">
+    <span class="a-credit-card-apr-rating--{{ adjective }}">
         {%- for i in range(card.purchase_apr_for_tier_rating + 1) %}
             {{ svg_icon("dollar-round") }}
         {% endfor -%}
 
-        <strong>Pay {{ adjective }} interest</strong>
+        Pay {{ adjective }} interest
     </span>
 {%- endmacro -%}
 
-{%- set card_columns = [
-    {'heading': 'Credit card'},
-    {'heading': 'Purchase APR'},
-    {'heading': 'Account fee'},
-    {'heading': 'Balance transfer APR'},
-    {'heading': 'Offers rewards'},
-] %}
-{%- set card_rows = [] %}
-{%- for card in results %}
-    {% set rewards = [] %}
-    {% for reward in card.rewards %}
-        {% do rewards.append(rewards_lookup[reward]) %}
-    {% endfor %}
+{%- macro card_rewards(card) -%}
+    {%- set rewards = [] -%}
+    {%- for reward in card.rewards -%}
+        {%- do rewards.append(rewards_lookup[reward]) -%}
+    {%- endfor -%}
+    {{- (rewards | join(', ')) if rewards else 'None' -}}
+{%- endmacro -%}
 
-    {% do card_rows.append( [
-        card_name_cell(card) | safe,
-        purchase_apr_cell(card) | safe,
-        (card.periodic_fee_type | join(', ')) if card.periodic_fee_type else 'None',
-        apr(card.transfer_apr_for_tier) if card.transfer_apr_for_tier is not none else apr_range(card.transfer_apr_min, card.transfer_apr_max),
-        (rewards | join(', ')) if rewards else 'None'
-    ] ) %}
-{% endfor %}
-{%- with value = {
-    'data': {
-        'columns': card_columns,
-        'rows': card_rows
-    },
-    'options': ['directory_table']
-} %}
-    {% include 'v1/includes/organisms/tables/base.html' %}
-{% endwith %}
+<div class="o-card-group">
+    <div class="o-card-group_cards">
+        {%- for card in results %}
+        <article class="m-card m-card--tabular">
+            <a href="{{ card.url }}">
+                <hgroup class="m-card__heading-group">
+                    <h3 class="m-card__heading">{{ card.institution_name | safe }}</h3>
```

Comment:
```suggestion
                    <h3 class="m-card__heading">{{ card.institution_name }}</h3>
```

Why does this need a `| safe`? If for some reason an institution had HTML tags in its name we'd have a bad time.

Reply:
My mom always said better to be safe than sorry